index.rst

```diff
@@ -13,6 +13,7 @@ Contents:
 
 .. toctree::
    :maxdepth: 2
+   :caption: Graylog Server
```

Comment:
Maybe call this caption simply Graylog? In the end we describe more than just the server (web interface, and some plugins)